ibis/backends/tests/test_generic.py

```diff
@@ -916,6 +916,46 @@ def test_memtable_bool_column(backend, con, monkeypatch):
     backend.assert_series_equal(t.a.execute(), pd.Series([True, False, True], name="a"))
 
 
+@pytest.mark.notimpl(
+    ["datafusion"], reason="no memtable support", raises=com.OperationNotDefinedError
+)
+@pytest.mark.broken(
+    ["druid"], reason="returns empty string for None", raises=AssertionError
+)
+def test_memtable_null_str_default_dtype(con):
+    """Ensure that null strings are translated to some kind of null value."""
+    t = ibis.memtable(pd.DataFrame({"a": ["a", None, "c"]}))
+    result = con.execute(t.a)
+    assert result.iat[0] == "a"
+    assert pd.isna(result.iat[1])
```

Comment:
why the general `pd.isna`? What is stopping us from doing `assert result.iat[1] is None`? Adding a comment explaining would be nice

Reply:
Some backends (it may just be impala, but I don't really want to fix that right now) do not return `None` and return `np.nan`.